src/mazela-artur/Fsd.Artur.Cs3/Fsd.Artur.Cs3/ShopActions/AddProducts.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Fsd.Artur.Cs3.WareHouse;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Artur.Cs3.ShopActions
+{
+    public class AddProducts
```

Comment:
błędne podejście do nazewnictwa klas i metod, na to już w tym zadaniu nie zwracam uwagi bo pewnie jest w innych miejscach tak samo